type/__apt_source/manifest

```diff
@@ -31,17 +31,23 @@ fi
 component="$(cat "$__object/parameter/component")"
 
 if [ -f "$__object/parameter/arch" ]; then
-   forcedarch="[arch=$(cat "$__object/parameter/arch")]"
-else
-   forcedarch=""
+	forcedarch="arch=$(cat "$__object/parameter/arch")"
+fi
+
+if [ -f "$__object/parameter/signed-by" ]; then
+   signed_by="${signed_by:${signed_by} }signed-by=$(cat "$__object/parameter/signed-by")"
```

Comment:
You need to join the lines with a comma.

e.g. `sed -e ':a' -e '$!N' -e 's/\n/,/' -e 'ta'` instead of `cat`

cf. also [apt/methods/gpgv.cc:497](https://salsa.debian.org/apt-team/apt/-/blob/2.5.1/methods/gpgv.cc#L497)